### DISCUSSION TITLE
VUE-CESIUM GLOBE IS NOT SHOWING..

### DISCUSSION THREAD

**Discussion by DSoftwareArtist** [zouyaoji/vue-cesium#243] [Q&A] [answered]
Hello all,
I am not sure if you are already into discussion of this issue. Vue-Cesium globe is not showing.. I have the impression that this was because of the [recent release](https://cesium.com/blog/2022/08/01/cesium-releases-in-august-2022). 

https://zouyaoji.top/vue-cesium/#/en-US/component/vc-viewer
![Screenshot from 2022-08-02 15-14-15](https://user-images.githubusercontent.com/13848044/182314687-c75fb9bb-0e7b-4c91-bf6a-f1208dc04da3.png)

Is there something I can do to make my project work again? Big Thanks!

**Comment by whyds000** [comment, marked as answer]
Me too, it was able to show yesterday, but not today

**Reply by whyds000** [reply]
and I did not upgrade the version

**Reply by zouyaoji** [reply]
It may be that the 1.96 version of Cesium has a disruptive update, causing vue-cesium to be abnormal, and it is being repaired. Before repairing, please specify the version of CesiumJS as the previous version through cesiumPath.

**Reply by DSoftwareArtist** [reply]
From `"vue-cesium": "^3.0.14"`, I already upgraded to `"vue-cesium": "^3.0.15"`.  But still not working.

**Reply by zouyaoji** [reply]
app.use(VueCesium, {
  cesiumPath: 'https://unpkg.com/cesium@1.95/Build/Cesium/Cesium.js'
})

**Reply by DSoftwareArtist** [reply]
Thanks! That helps. Its working again.

**Reply by zouyaoji** [reply]
The 3.0.16 version can works in CesiumJS1.96.
